+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'start to run test.'
+ message='start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start to run test.'
Fri Sep  1 11:38:11 2023 - INFO  - start to run test.
+ lsscsi -g
++ wc -l
+ test 1 -eq 0
+ CHECK_RESULT 1 0 0 'show SCSI universal device file names fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='show SCSI universal device file names fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'show SCSI universal device file names fail'
+ LOG_ERROR 'show SCSI universal device file names fail'
+ message='show SCSI universal device file names fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'show SCSI universal device file names fail'
Fri Sep  1 11:38:12 2023 - ERROR - show SCSI universal device file names fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsscsi_1.sh line 25'
+ message='oe_test_lsscsi_1.sh line 25'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsscsi_1.sh line 25'
Fri Sep  1 11:38:12 2023 - ERROR - oe_test_lsscsi_1.sh line 25
+ return 0
+ lsscsi -k
++ wc -l
+ test 1 -eq 0
+ CHECK_RESULT 1 0 0 'show kernel name fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='show kernel name fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'show kernel name fail'
+ LOG_ERROR 'show kernel name fail'
+ message='show kernel name fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'show kernel name fail'
Fri Sep  1 11:38:13 2023 - ERROR - show kernel name fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsscsi_1.sh line 28'
+ message='oe_test_lsscsi_1.sh line 28'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsscsi_1.sh line 28'
Fri Sep  1 11:38:13 2023 - ERROR - oe_test_lsscsi_1.sh line 28
+ return 0
+ lsscsi -d
++ wc -l
+ test 1 -eq 0
+ CHECK_RESULT 1 0 0 'show the primary and secondary numbers of device fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='show the primary and secondary numbers of device fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'show the primary and secondary numbers of device fail'
+ LOG_ERROR 'show the primary and secondary numbers of device fail'
+ message='show the primary and secondary numbers of device fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'show the primary and secondary numbers of device fail'
Fri Sep  1 11:38:14 2023 - ERROR - show the primary and secondary numbers of device fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsscsi_1.sh line 31'
+ message='oe_test_lsscsi_1.sh line 31'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsscsi_1.sh line 31'
Fri Sep  1 11:38:14 2023 - ERROR - oe_test_lsscsi_1.sh line 31
+ return 0
+ lsscsi -H
++ wc -l
+ test 1 -eq 0
+ CHECK_RESULT 1 0 0 'show the SCSI host currently connected to the system fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='show the SCSI host currently connected to the system fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'show the SCSI host currently connected to the system fail'
+ LOG_ERROR 'show the SCSI host currently connected to the system fail'
+ message='show the SCSI host currently connected to the system fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'show the SCSI host currently connected to the system fail'
Fri Sep  1 11:38:14 2023 - ERROR - show the SCSI host currently connected to the system fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsscsi_1.sh line 34'
+ message='oe_test_lsscsi_1.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsscsi_1.sh line 34'
Fri Sep  1 11:38:15 2023 - ERROR - oe_test_lsscsi_1.sh line 34
+ return 0
+ lsscsi -l
++ wc -l
+ test 1 -eq 0
+ CHECK_RESULT 1 0 0 'show the additional information for each SCSI device fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='show the additional information for each SCSI device fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'show the additional information for each SCSI device fail'
+ LOG_ERROR 'show the additional information for each SCSI device fail'
+ message='show the additional information for each SCSI device fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'show the additional information for each SCSI device fail'
Fri Sep  1 11:38:15 2023 - ERROR - show the additional information for each SCSI device fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsscsi_1.sh line 37'
+ message='oe_test_lsscsi_1.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsscsi_1.sh line 37'
Fri Sep  1 11:38:16 2023 - ERROR - oe_test_lsscsi_1.sh line 37
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Fri Sep  1 11:38:16 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 5
+ test 5 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Sep  1 11:38:17 2023 - ERROR - failed to execute the case.
+ exit 5
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Sep  1 11:38:17 2023 - INFO  - start environment cleanup.
+ rm -rf /tmp/error.log
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Fri Sep  1 11:38:17 2023 - INFO  - Finish environment cleanup!
